modules/react/common/lib/utils/elements.ts

```diff
@@ -35,7 +35,11 @@ export const getFirstFocusableElement = (container: HTMLElement): HTMLElement |
 
   for (let i = 0; i < elements.length; i++) {
     const element = elements.item(i);
-    if (element && isFocusable(element as HTMLElement)) {
+    if (
+      element &&
+      isFocusable(element as HTMLElement) &&
```

Comment:
Do you still need to cast it as HTMLElement?

Reply:
No, we do not. Nice catch